pkg/controller/opentelemetrycollector/reconcile_deployment.go

```diff
@@ -83,6 +83,32 @@ func deployment(ctx context.Context) *appsv1.Deployment {
 		args = append(args, fmt.Sprintf("--%s=%s", k, v))
 	}
 
+	configMapVolumeName := fmt.Sprintf("reserved-%s", name)
```

Comment:
Curious why the `reserved-` prefix? Is this name meaningful enough - should it be more opentelemetry specific?

Reply:
This is an "internal" name, to avoid getting into a conflict between the config map we create with the OpenTelemetry Collector configuration and a similarly named volume defined by the user. I first tried adding a couple of underscores in the name, but that's not allowed :-/

I'm not happy with this prefix and would appreciate suggestions!

Reply:
Maybe `otc-internal-` or variations, `otel-collector-`?

Reply:
`otc-internal` sounds good!